etl/steps/data/garden/bp/2023-02-20/shared.py

```diff
@@ -0,0 +1,106 @@
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+from structlog import get_logger
+
+from etl.data_helpers import geo
+
+log = get_logger()
+
+CURRENT_DIR = Path(__file__).parent
+VERSION = CURRENT_DIR.name
+
+
+def add_population(
```

Comment:
Could this be somewhere in `etl.data_helpers`? Looks pretty useful for other datasets too.

Reply:
Thanks. I think it would be good to give this function a bit of extra thought before sharing it in data helpers, to avoid what happened in the past with datautils.geo functions, so I'll leave it for a different PR.